cpp/src/test/autogenerated/cmap_basic_test.cc

```diff
@@ -67,11 +67,11 @@ void CMapBasicTests::SetUp() {
   LoadFont(font_name.c_str(), font_factory, &font_array);
   ASSERT_FALSE(font_array.empty());
   Ptr<Font> font = font_array.at(0);
-  ASSERT_NE(font, reinterpret_cast<Font*>(NULL));
+  ASSERT_NE(font, static_cast<Font*>(NULL));
```

Comment:
I believe all the ASSERT/EXPECT EQ/NE checks can just take a nullptr, with no casting.